.github/actions/verify-compose/docker_compose.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash -e
+
+if [ -z "${ACS_VERSION}" ]; then
+  echo "ACS_VERSION variable is not set"
+  exit 2
+fi
+if [ -z "${COMMIT_MESSAGE}" ]; then
+  echo "COMMIT_MESSAGE variable is not set"
+  exit 2
+fi
+if [ -z "${BRANCH_NAME}" ]; then
+  echo "BRANCH_NAME variable is not set"
+  exit 2
+fi
+
+GIT_DIFF=$(git diff origin/master --name-only .)
+compose_file="docker-compose.yml"
+alf_port=8080
+
+if [[ ${ACS_VERSION} != "latest" ]]; then
+  export compose_file="${ACS_VERSION}-docker-compose.yml"
```

Comment:
maybe change the input of the action and expect to provide a full path to the docker compose file to test

Reply:
I did that, now the action verify-compose expects full path to the docker compose file. Run on which it worked: https://github.com/Alfresco/acs-deployment/actions/runs/2838100486